msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerBaseStrategy.java

```diff
@@ -79,53 +94,16 @@ Handler getPreferredHandler() {
         }
     }
 
-    Bundle getSilentBrokerRequestBundle(final AcquireTokenSilentOperationParameters parameters) {
-        final MsalBrokerRequestAdapter msalBrokerRequestAdapter = new MsalBrokerRequestAdapter();
-
-        final Bundle requestBundle = new Bundle();
-        final BrokerRequest brokerRequest = msalBrokerRequestAdapter.
-                brokerRequestFromSilentOperationParameters(parameters);
-
-        requestBundle.putString(
+    protected Intent completeInteractiveRequestIntent(@NonNull final Intent interactiveRequestIntent,
+                                                      @NonNull final AcquireTokenOperationParameters parameters){
+        interactiveRequestIntent.putExtra(
                 AuthenticationConstants.Broker.BROKER_REQUEST_V2,
-                new Gson().toJson(brokerRequest, BrokerRequest.class)
-        );
-
-        requestBundle.putInt(
-                AuthenticationConstants.Broker.CALLER_INFO_UID,
-                Binder.getCallingUid()
+                new Gson().toJson(
+                        mRequestAdapter.brokerRequestFromAcquireTokenParameters(parameters),
+                        BrokerRequest.class)
         );
+        interactiveRequestIntent.putExtra(AuthenticationConstants.Broker.ACCOUNT_NAME, parameters.getLoginHint());
```

Comment:
is this new ? why are we passing this seperately, wouldn't this be part of the `BrokerRequest`?

Reply:
Seems like the only place that uses this (apart from logging) is BrokerAccountService - meaning this is not needed.